numpy/ma/core.pyi

```diff
@@ -550,7 +552,33 @@ class _extrema_operation(_MaskedUFunc):
     def reduce(self, target, axis=...): ...
     def outer(self, a, b): ...
 
-def min(obj, axis=..., out=..., fill_value=..., keepdims=...): ...
+@overload
+def min(
+    obj: _ArrayLike[_SCT],
+    axis: None = ...,
+    out: None = ...,
+    fill_value: _ScalarLike_co | None = ...,
+    keepdims: Literal[False] = ...,
```

Comment:
I realize that this isn't done a lot in numpy at the moment, but it can be helpful to also include the default values in the signatures. See e.g. https://github.com/numpy/numpy/issues/28428 

```suggestion
    axis: None = None,
    out: None = None,
    fill_value: _ScalarLike_co | None = None,
    keepdims: Literal[False] | _NoValueType = ...,
```

The `_NoValueType` has yet to be imported from `numpy._globals`. It's an annoying one to type, but I know from numtype that `stubtest` demands that we do.